src/game/MatrixGeneration.ts

```diff
@@ -0,0 +1,249 @@
+import GameConfiguration from "./GameConfiguration";
+
+import seedrandom from "seedrandom";
+
+//TODO should this defined here?
+//TODO how to choose these numbers?
+//TODO is this correct way to provide constants? (what are the naming conventions?)
+const MinMatrixSize = 3;
```

Comment:
Use lowercase. Location is actually fine.

Reply:
Resolved in https://github.com/Kugelschieber/breach/pull/22/commits/b5d06afc7b3106e14a489c11bd895284e499dda6